components/Chips/src/private/MDCChipView+Testing.h

```diff
@@ -0,0 +1,24 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCChipView.h"
+
+/**
+ Exposes parts of MDCChipView for testing.
+ */
+@interface MDCChipView (Testing)
+
+- (void)updateTitleFont;
```

Comment:
instead of exposing this API it will be more future proof to post the same notification that the system does.

NotificationCenter.default.post(Notification(name: .FontMetricsContentSizeCategoryDidChange))
which i found on https://medium.com/livefront/practical-dynamic-type-part-2-testing-613bb845f26b

Reply:
Good point! Addressed in 7bfe32288e481a9ba7b7e16cd234acffc235f035.